core/config.js

```diff
@@ -123,9 +123,9 @@ exports.loadOutputConfig = function (outputName) {
     if (!config[outputName]) {
         config[outputName] = {};
     }
-	
+
     for (var obj in config) {
-        if (modes.includes(obj)) continue;
+        if (integs.find(int => int.name === obj)) continue;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'continue'.](https://www.codacy.com/app/theprojectshop/Kassy/file/2530929716/issues/source?bid=3282987&fileBranchId=3283570#l128)